src/VaultSharp/V1/SecretsEngines/PKI/AbstractCertificateRequestOptions.cs

```diff
@@ -0,0 +1,113 @@
+using Newtonsoft.Json;
+
+namespace VaultSharp.V1.SecretsEngines.PKI
+{
+    /// <summary>
+    /// Represents the Certificate request options.
+    /// </summary>
+    public abstract class AbstractCertificateRequestOptions
```

Comment:
Can you please replicate the class for your need and leave the original POCO intact?

For payload classes, I am deliberately avoiding inheritance to decouple them.